Networking/Networking/Model/StateOfACountry.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Codegen
+
+/// Represents a State Entity within a Country.
+///
+public struct StateOfACountry: Decodable, Equatable, GeneratedFakeable {
+    public let code: String
```

Comment:
Optional. Add an example as a comment, e.g.:

// E.g., ON for Ontario. Note: Not always two letters.